portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverviewAttributeBreakdown.testcase

```diff
@@ -105,14 +109,14 @@ definition {
 
 		task ("Filter Attribute by Boolean identifier in Overview") {
 			ACSegments.addBreakdownByAttribute(
-				attribute = "doNotCall",
+				attribute = "User Custom Field",
 				breakdownName = "Segment Overview Test");
 		}
 
 		task ("View Breakdown Chart") {
 			ACIndividualsDashboard.viewBreakdownRecharts(
 				fillColor = "#4B9BFF",
-				tick = "false");
+				tick = "true");
```

Comment:
Hey @manuferreira, I think you missed putting this value as TRUE in the user account configuration in DXP or is this not necessary?

Reply:
Oh you mean, when I'm adding the custom attribute for the user I should set it to True?
The default value for the custom attribute that I created is True, so no need to set it again in the user configuration.

Reply:
Ohhh ok, thanks for the explanation, Manu 😄